discovery-infra/test_infra/utils.py

```diff
@@ -620,3 +620,19 @@ def config_etc_hosts(cluster_name: str, base_dns_domain: str, api_vip: str):
         hosts_lines.append(f"{api_vip} {api_vip_dnsname}\n")
     with open("/etc/hosts", "w") as f:
         f.writelines(hosts_lines)
+
+def run_container(container_name, image ,flags=[]):
+    logging.info(f'Running Container {container_name}')
+    run_container_cmd = f'podman run  {consts.PODMAN_FLAGS} --name {container_name}'
+    
+    for flag in flags:
+        run_container_cmd += f' {flag}'
+
+    run_container_cmd += f' {image}'
+
+    os.system(run_container_cmd)
+
+def remove_running_container(container_name):
+    logging.info(f'Removing Container {container_name}')
+    container_rm_cmd = f'podman stop {container_name} {consts.PODMAN_FLAGS} && podman rm {container_name} {consts.PODMAN_FLAGS}'
```

Comment:
Please set `{consts.PODMAN_FLAGS}` before the podman operation and `container_name` for clarity.

Reply:
changed